internal/config/config.go

```diff
@@ -27,6 +27,69 @@ type ConfigFile struct {
 	Project         string `json:"project,omitempty" yaml:"project,omitempty"`
 }
 
+type Config struct {
```

Comment:
This is a new type that encapsulates updating the config file. It will eventually replace the `ConfigFile` type.